src/main/java/wolox/training/models/Book

```diff
@@ -0,0 +1,128 @@
+package wolox.training.models;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
```

Comment:
If the only thing you want is the nullable = false, you can use directly the javax validation annotation "@NotNull".
@Column annotation is not necessary at all with that.

Reply:
ok in the other cases, when I don't have the "@NotNull" annotation it must be the "@Column" annotation in there instead, right?

Reply:
Nope, it's not necessary unless you want to set an attribute. column name is inferred from variable name, and all are mapped. The modifier you would use to tell a field not to be mapped to the DB is `transient`... I believe it was `private transient Integer variableName` for example

Reply:
ok, thank you